packages/driver/src/cross-origin/patches/fetchAndXMLHttpRequest.ts

```diff
@@ -0,0 +1,102 @@
+const captureFullRequestUrl = (relativeOrAbsoluteUrlString, window) => {
```

Comment:
nit: thought on making these two files to name `patches/fetch` and `patchees/xmHttpRequest`? Could import into an extend file sot here's only one import in cross-origin/cypress?

Reply:
I originally had  them as two separate files, but since now both use `captureFullRequestUrl` as a utility, I wasn't sure on a good location to put the utility function since it isn't patching anything. I think if we find an appropriate spot I am completely open to moving these into a single files

Reply:
I'd just put a utils folder inside the patches folder since it's locally only used there.

Reply:
addressed in [2dc3592](https://github.com/cypress-io/cypress/pull/23822/commits/2dc3592616d53196a071b2dbcdb7746f8b1857fe)